README.md

```diff
@@ -39,7 +41,7 @@ An interactive voting wizard is provided for Ubuntu. After setting up a baking i
 on mainnet, you can vote by running:
 
 ```bash
-tezos-voting-wizard
+tezos-vote
 ```
 
```

Comment:
Likewise I think it's worth mentioning it here to show people at a glance what happened:
```suggestion
> :warning: Since [#570](https://github.com/serokell/tezos-packaging/pull/570) this wizard was renamed from `tezos-voting-wizard`

```